tests/csm/rest/test_csm_users.py

```diff
@@ -4743,6 +4745,117 @@ def test_44787(self):
+            assert init_token != new_token, "unexpected token mismatch"
+            # CORTX-33869 for != or ==
+
+        self.log.info("##### Test completed -  %s #####", test_case_name)
+
+
+    @pytest.mark.sanity
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.tags('TEST-44803')
+    def test_44803(self):
+        """
+        Verify different token is generated when user is recreated.
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+
+        user_roles = ["admin", "manage", "monitor"]
+        for role in user_roles:
+            self.log.info("Step 1: Create new user")
+            password = self.csm_conf["test_44803"]["current_password"]
+            response = self.csm_obj.create_csm_user(user_type="valid",
+                                                    user_role=role,
+                                                    user_password=password)
+            self.log.info("Verifying if user was created successfully")
+            assert response.status_code == const.SUCCESS_STATUS_FOR_POST
+            username = response.json()["username"]
+            user_id = response.json()["id"]
+            self.created_users.append(username)
+
+            self.log.info("Step 2: Login and get header")
+            init_header = self.csm_obj.get_headers(username, password)
+            self.log.info("Store Authorization Token")
+            init_token = init_header['Authorization']
+
+            self.log.info("Step 3: Sending request to delete csm user %s", username)
+            response = self.csm_obj.delete_csm_user(user_id)
+            self.log.info("Verify success response")
+            assert response.status_code == HTTPStatus.OK, "User Not Deleted Successfully."
+            self.created_users.remove(username)
+
+            self.log.info("Step 4: Recreate user with same creds")
+            password = self.csm_conf["test_44803"]["current_password"]
+            response = self.csm_obj.create_csm_user(user_type="valid",
+                                                    user_role=role,
+                                                    user_name=username,
+                                                    user_password=password)
+            self.log.info("Verifying if user was created successfully")
+            assert response.status_code == const.SUCCESS_STATUS_FOR_POST
+            username = response.json()["username"]
+            user_id = response.json()["id"]
+            self.created_users.append(username)
+
+            new_header = self.csm_obj.get_headers(username, password)
+            self.log.info("Store Authorization Token")
+            new_token = new_header['Authorization']
+
+            self.log.info("Check Bearer token should be different")
+            self.log.info("init_token = %s new_token = %s ", init_token, new_token)
```

Comment:
Can u make this info more clear
also, suggestion, it will be good if we can have two separate logs

Reply:
updated